storage/mongodb/platform_image.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2018 tsuru authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mongodb
+
+import (
+	"github.com/globalsign/mgo"
+	"github.com/globalsign/mgo/bson"
+	"github.com/tsuru/tsuru/db"
+	dbStorage "github.com/tsuru/tsuru/db/storage"
+	"github.com/tsuru/tsuru/types/app/image"
+)
+
+var _ image.PlatformImageStorage = &PlatformImageStorage{}
+
+type PlatformImageStorage struct{}
+
+type platformImage struct {
+	Name   string `bson:"_id"`
```

Comment:
I personally don't like using values like names as collection ids. Specially in cases like this, where it's the name of another entity (platform). I'd prefer using the platform name as a regular field, and creating an index for it.